### PR TITLE
RCHAIN-2747 remove faucet (part 1)

### PR DESCRIPTION
**Pull request by rabbitonweb**
## Overview
**Death to Faucet!**

This is part 1/2 of faucet removal. Should be straightforward, lines of code are being dropped.

### JIRA ticket:
https://rchain.atlassian.net/browse/RCHAIN-2747

### CODE REVIEW

**Comment by rabbitonweb** [comment]
bors try

**Comment by rabbitonweb** [comment]
This PR is ready for review

File: integration-tests/test/test_internal.py [hunk 1]
```diff
@@ -125,7 +124,6 @@ def test_parse_show_block_output() -> None:
...
...1} <- x7 ) {\n          @{x4}!((true, \"Bond successful!\")) |\n          @{Unforgeable(0x4de349932e7e2443526ad0a4923cd69c0027184659a6c475112fecca2008dedb)}!((x11).set(x0,((x2 - x8), x1, x3, ((x9).size() + 1))))\n        }\n      }\n    } |\n    for( @{x8}, @{x9}, @{x10}, @{x11} <= x6 ) {\n      match x8 {\n        {x13 : (x14, x15, x16, x17)...x12} => x6!(x12, x9, (x10).set(x13,((x14 + (x9).nth((x17 - 1))), x15, x16, x17)), x11) ;\n        _ => @{x11}!(x10)\n      }\n    }\n  }\n} |\nfor( @{x0}, @{x1}, @{x2} <= @{Unforgeable(0x39bdf789fb12e1c3487abce560193e3a9545d2d7e311aca70f6f47d36ba5115a)} ) {\n  new x3, x4, x5, x6, x7, x8 in {\n    @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"range\", 1, (x0 + 1), *x3) |\n    for( @{x9} <- x3 ) {\n      @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"map\", x9, *x4, *x5) |\n      for( @{x10} <- x5 ) {\n        @{bundle+ {   Unforgeable(0x6df7905a81f51e556695f3cbeccd1a81ffa1f3ab4c0884d18324d709cc8ea406) }}!(\"fold\", x10, 0, *x6, *x8) |\n        for( @{x11} <- x8 ) {\n          @{x2}!(([(((x10).nth(0) + x1) - x11)] ++ (x10).slice(1,x0)))\n
```
Comment (KentShikama) [inline integration-tests/test/test_internal.py]: These tupleSpaceDump changes shouldn't be relevant

Comment (rabbitonweb) [reply integration-tests/test/test_internal.py]: they had faucet in the dump, also they are not actually in the show_block output (why assert works I have no idea :) )

**Pull request merged**